config/vector.toml

```diff
@@ -9,19 +9,40 @@
 # ------------------------------------------------------------------------------
 # Website: https://vector.dev
 # Docs: https://vector.dev/docs/
+# Chat: https://chat.vector.dev
 # ------------------------------------------------------------------------------
 
-# Note: A full configuration reference is located at https://vector.dev/docs/ and examples
-#       in the ./examples folder.
+# data_dir = "/var/lib/vector"
 
-data_dir = "/var/lib/vector"
+# Vector's API for introspection
+# Try it out with the `vector top` command
+[api]
+enabled = true
+address = "127.0.0.1:8686"
 
-# Input data. Change me to a valid input source.
-[sources.in]
-  type = "stdin"
+# Dummy Apache formatted logs
+[sources.dummy_logs]
+type = "generator"
+format = "apache_common"
+interval = 1
 
-# Output data
-[sinks.out]
-  inputs   = ["in"]
-  type     = "console"
-  encoding = "text"
+# Parse Apache formatted logs
+# See the Remap reference: https://vector.dev/docs/reference/remap/
+[transforms.parse_logs]
+type = "remap"
+source = '''
+. = parse_apache_log(.message)
```

Comment:
As we don't currently have this function, I'd recommend using a Syslog example instead